browser/src/control/Control.StatusBar.js

```diff
@@ -194,7 +194,13 @@ class StatusBar extends JSDialog.Toolbar {
 			],
 			vertical: false,
 			visible: false
-		};
+		}
+
+		if (dataPriority) {
+			item['data-priority'] = dataPriority;
```

Comment:
can we avoid `-` ? (camelCase instead)
It will be less problematic later, eg when using as `item.dataPriority`